packages/nextra-theme-docs/src/utils/normalize-pages.ts

```diff
@@ -9,18 +9,10 @@ function extendMeta(
   if (typeof meta === 'string') {
     meta = { title: meta }
   }
-  const theme = Object.assign({}, fallback.theme, meta.theme)
+  const theme: PageTheme = Object.assign({}, fallback.theme, meta.theme)
   return Object.assign({}, fallback, meta, { theme })
 }
 
-/**
- * An option to control how an item should be displayed in the sidebar:
- * - `normal`: the default behavior, item will be displayed
- * - `hidden`: the item will not be displayed in the sidebar entirely
- * - `children`: if the item is a folder, itself will be hidden but all its children will still be processed
- */
-export type Display = 'normal' | 'hidden' | 'children'
-
 export interface Item extends MdxFile {
```

Comment:
what is `Item` and which `type` should it have? Since we have `PageItem` that should have `type: page` and DocsItem with `type: doc` (that is by default). I am only confused by `interface Item` 😅